spec/factories.rb

```diff
@@ -13,4 +13,44 @@
   factory :game do
     association :user_white, factory: :user
   end
+
+  factory :piece do
+    x_position 1
+    y_position 1
+  end
+end
+
+# I want this method to populate a board with the obstructed pieces that are in Ken's example
+def obstructed_game
+  obstructed_game = FactoryGirl.create(:game)
+  piece_a1 = FactoryGirl.create(:piece, game_id: obstructed_game.id, x_position: 1, y_position: 1)

```

Comment:
I made both changes. I did away with the variable assignments as they weren't necessary, and moved obstructed_game to piece_spec.rb.